.dev/log_collector/example_config.py

```diff
@@ -0,0 +1,16 @@
+work_dir = '../mmsegmentation-master/work_dirs'
+metric = 'mIoU'
+# Don't specify the log_items and ignore_keywords at the same time
+log_items = [
+    'segformer_mit-b5_512x512_160k_ade20k_cnn_lr_with_warmup',
+    'segformer_mit-b5_512x512_160k_ade20k_cnn_no_wramup_lr',
+    'segformer_mit-b5_512x512_160k_ade20k_mit_trans_lr',
+    'segformer_mit-b5_512x512_160k_ade20k_swin_trans_lr'
+]
+# or
+ignore_keywords = ['segformer']
```

Comment:
if `log_items` and `ignore_keywords` cannot be specified at the same time, I suggest:
```suggestion
# or only set ignore_keywords, here is an example
# the folder name containing `'segformer'` won't be collected
# ignore_keywords = ['segformer']
```